ci/do_ci.sh

```diff
@@ -92,19 +105,31 @@ if [[ "$1" == "bazel.release" ]]; then
 elif [[ "$1" == "bazel.release.server_only" ]]; then
   setup_clang_toolchain
   echo "bazel release build..."
-  bazel_release_binary_build
+  bazel_binary_build release
+  exit 0
+elif [[ "$1" == "bazel.sizeopt.server_only" ]]; then
+  setup_clang_toolchain
+  echo "bazel size optimized build..."
+  bazel_binary_build sizeopt
+  exit 0
+elif [[ "$1" == "bazel.sizeopt" ]]; then
+  setup_clang_toolchain
+  echo "bazel size optimized build with tests..."
+  bazel_binary_build sizeopt
+  echo "Testing..."
+  bazel test ${BAZEL_TEST_OPTIONS} //test/... --config optimize_binary_size=enabled
```

Comment:
This is the old version. Update or delete this section.

Reply:
Ah, thanks.